parser-typechecker/tests/Unison/Test/TermPrinter.hs

```diff
@@ -160,13 +160,13 @@ test = scope "termprinter" . tests $
   , pending $ tc "case e of { () -> k } -> z" -- TODO doesn't parse since 'many leaf' expected before the "-> k"
                                               -- need an actual effect constructor to test this with
   , tc "if a then if b then c else d else e"
-  , tc "handle foo in (handle bar in baz)"
+  , tc "handle handle foo with bar with baz"
```

Comment:
stacked handler syntax is kinda weird after the patch... As a user, I'd probably use parens.

Reply:
It looks ok to me. Not unlike `if x then if y then z else p else q`